### PR TITLE
local-volume template

### PR DESCRIPTION
**Pull request by nkkashyap**
Signed-off-by: Neeraj K Kashyap <nkashyap@in.ibm.com>
GHE Issue: https://github.com/IBM/ibm-satellite-storage/issues/17
Verification Test: https://github.com/IBM/ibm-satellite-storage/issues/17#issuecomment-760178811

### CODE REVIEW

**Comment by nkkashyap** [comment]
GHE Issue: https://github.com/IBM/ibm-satellite-storage/issues/17

File: config-templates/redhat/local-volume/4.5/custom-parameters.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "description": "Node label key",
+    "displayname": "Node Label Key",
+    "name": "label-key",
+    "required": true,
```
Comment (prgavali) [inline config-templates/redhat/local-volume/4.5/custom-parameters.json:6]: Add all `true`/`false` in the double quotes

**Review by prgavali — CHANGES_REQUESTED** [review]
reviewed

File: config-templates/redhat/local-volume/4.5/custom-parameters.json (hunk 1 above)
Comment (nkkashyap) [reply config-templates/redhat/local-volume/4.5/custom-parameters.json]: replaced all `"true"`/`"false"` with `"true"`/`"false"`

File: config-templates/template_list.json [hunk 2]
```diff
@@ -22,5 +22,13 @@
...
     "enabled": true,
     "provider": "ibm"
+  },
+  {
+    "name": "local-volume",
+    "displayname": "Persistent storage using local volumes",
+    "Description": "Persistent storage using local volumes for satallite cluster",
+    "default-version": "4.5",
+    "enabled": true,
+    "provider": "redhat"
```
Comment (prgavali) [inline config-templates/template_list.json:32]: Add all boolean in double quote

File: config-templates/template_list.json [hunk 3]
```diff
@@ -22,5 +22,13 @@
     "default-version": "4.3",
     "enabled": true,
```
Comment (prgavali) [inline config-templates/template_list.json:23]: Add all boolean in double quote

**Review by prgavali — CHANGES_REQUESTED** [review]
added comments

File: config-templates/template_list.json (hunk 2 above)
Comment (nkkashyap) [reply config-templates/template_list.json:32]: Done

**Review by prgavali — APPROVED** [review]
LGTM

**Pull request merged**